GVFS/GVFS.Hooks/HooksPlatform/GVFSHooksPlatform.Mac.cs

```diff
@@ -1,42 +1,42 @@
-using GVFS.Platform.Mac;
+using GVFS.Platform.POSIX;

```

Comment:
One additional thought here is that we should presumably also rename this file to `GVFSHooksPlatform.POSIX.cs` and adjust the various `*.csproj` files to use that instead.

Reply:
There'll likely need to be one each for Mac and Linux since `GetInstallerExtension` will differ for both. It's a bit awkward; any thoughts? Could dump everything except `GetInstallerExtension` into `GVFSHooksPlatform.POSIX`, make the class partial, and then have a `.Mac` and `.Linux` variant with only `GetInstallerExtension`?

Reply:
> Could dump everything except GetInstallerExtension into GVFSHooksPlatform.POSIX, make the class partial, and then have a .Mac and .Linux variant with only GetInstallerExtension?

That sounds good to me.  I am okay with leaving the current code as-is and making that change in a follow up PR if you prefer.

Reply:
@kivikakk I just realized nothing is calling `GetInstallerExtension` anymore.  Could you just remove it from this class (and the .Windows.cs version of this class)?